conf/dias.conf

```diff
@@ -9,7 +9,13 @@
 
```

Comment:
Why is this file both in ch_ansible and here? Are we using this one at all?

Reply:
It's probably my fault.  This one is here is so you can run dias locally while building analysers without having to install it.    The one in `ch_ansible` is specific to the deployment on `marimba`.

Reply:
@ketiltrout is right! This is for tests and running locally.